rpcs3/Emu/Cell/PPULLVMRecompiler.cpp

```diff
@@ -289,12 +289,20 @@ RecompilationEngine::RecompilationEngine()
 	, m_currentId(0)
 	, m_last_cache_clear_time(std::chrono::high_resolution_clock::now())
 	, m_compiler(*this, CPUHybridDecoderRecompiler::ExecuteFunction, CPUHybridDecoderRecompiler::ExecuteTillReturn, CPUHybridDecoderRecompiler::PollStatus) {
+
+	// PS3 has 256 Mb of memory.

```

Comment:
Totally unrelated. Yes, PS3 has something about 256+256 MB of *physical* memory. But we don't emulate physical memory, we emulate virtual memory which overall range is assumed to be 4 GB.

Here is 512 MB of the host memory allocated, which is quite a big amount, since most of it will be unused anyway. It approximately doubles rpcs3 memory consumption.